extensions/authentication-passport/docs.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
No need to have this file any more: `docs.json` is only used by `strong-docs`.

Reply:
good catch :)